cmd/helper.go

```diff
@@ -209,7 +209,14 @@ func doGet(command *cobra.Command, args []string) error {
 			return err
 		}
 
-		rsp.Password = body["accessToken"]
+		if v, ok := body["tokenType"]; ok && v == "TOKEN_TYPE_BEARER" {
```

Comment:
Are you sure it is this string constant?

Reply:
yes, this is the [value](https://github.com/calculi-corp/api/blob/761805bcdf25475af2e05ba0ac3a522be56897e4/proto/auth/credential.proto#L66)